docs/installation/migration-guide-from-dektrium-tools.md

```diff
@@ -11,6 +11,11 @@ composer remove dektrium/yii2-user
 composer remove dektrium/yii2-rbac
 ```
 
+## Install yii2-usuario
+```
+composer add 2amigos/yii2-usuario
```

Comment:
Shouldn't be `composer require 2amigos/yii2-usuario`?

Reply:
Yes it should be require.